Cargo.toml

```diff
@@ -54,6 +54,7 @@ features = [ "with-time", "with-rustc-serialize" ]
 
 [dev-dependencies]
 once_cell = "1.2.0"
+kuchiki = "0.7"
```

Comment:
Was it a deliberate decision to take 0.7? The [latest release is 0.8.0](https://crates.io/crates/kuchiki/0.8.0) and it seems to work on my machine.

Reply:
Whoops, good catch!